src/main/scala/outwatch/dom/dsl.scala

```diff
@@ -11,12 +12,14 @@ object dsl extends Attributes with Tags with Styles {
     object events extends Events
     object outwatch extends OutwatchAttributes
     object lifecycle extends OutWatchLifeCycleAttributes
+    object svg extends AttrsSvg
   }
+  object svg extends AttrsSvg with TagsSvg
```

Comment:
Not sure whether we need both `dsl.attributs.svg` and `dsl.tags.svg` as well as `dsl.svg`.